metagraph/wrappers.py

```diff
@@ -148,3 +149,86 @@ def assert_equal(
 
 class BipartiteGraphWrapper(Wrapper, abstract=BipartiteGraph, register=False):
     pass
+
+
+class NodeEmbeddingWrapper(Wrapper, abstract=NodeEmbedding, register=False):
+    def __init__(self, matrix, nodes=None):
+        super().__init__()
+        self.matrix = matrix
+        self.nodes = nodes
+
+    class TypeMixin:
+        _matrix_prop_map = {
```

Comment:
This is probably overkill if we only have one property (dtype of the matrix).

Reply:
https://github.com/metagraph-dev/metagraph/pull/133/commits/eb0eda1c191b9170b685a88b55e1b3319b42d464 refactors `NodeEmbeddingWrapper.TypeMixin` to implement this suggestion.